tests/cheetah/remove_call_test.py

```diff
@@ -23,7 +23,7 @@ def test_can_remove_calls(example, output):
     example = parse(example)
 
     calls = example.find_calls('foo')
-    assert len(calls) == 5
```

Comment:
I added more calls to one of the test inputs.  This was asserting that each of the input files had exactly 5 `$foo` calls, which seemed a little silly to me